### ISSUE TITLE
Support setting image for G4 Doorbell

### ISSUE THREAD

**Issue by cryptk** [uilibs/uiprotect#341]
### Is your feature request related to a problem? Please describe.

I have a custom background image that I like to use for my G4 Doorbell Pro.  I would like to be able to display text (for example "Hello cryptk!" when I scan my fingerprint and then some time later have it revert back to my custom image, but currently there is no support for setting the screen to show an image within uiprotect.

### Describe alternatives you've considered

I haven't been able to come up with any alternatives other than not using custom images

### Additional context

This is functionality that should eventually be exposed in Home Assistant, but the first step would be getting support in this library

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Are you willing to resolve this issue by submitting a Pull Request?

- [ ] Yes, I have the time, and I know how to start.
- [ ] Yes, I have the time, but I don't know how to start. I would need guidance.
- [X] No, I don't have the time, although I believe I could do it if I had the time...
- [ ] No, I don't have the time and I wouldn't even know how to start.

**Comment by RaHehl** [comment]
@cryptk Everything runs via `LCDMessage`, which can also have the `image` type. It is therefore already built into the library. If you’re missing that function in HA, you’d need to create a feature request there.

**Comment by RaHehl** [comment]
https://github.com/uilibs/uiprotect/blob/main/src/uiprotect/data/devices.py#L946
https://github.com/uilibs/uiprotect/blob/main/src/uiprotect/data/devices.py#L490
https://github.com/uilibs/uiprotect/blob/main/src/uiprotect/data/types.py#L412

**Comment by cryptk** [comment]
While there is an image message type, from looking through the code, I'm not understanding how to set the doorbell to show an image. I also haven't been able to find an example of how to do this in the documentation anywhere, and the CLI help text isn't offering any clues either. 

The library does properly return that the doorbell is displaying an image when it is set to an image, but as far as I can tell, currently the only way to set that image is through unifi protect. 

Could you give me a couple of clues on how to use this library to push an image to the display on the doorbell?

**Comment by RaHehl** [comment]
If you set a image via protect ui the api call looks like this: 
![image](https://github.com/user-attachments/assets/ff143de6-3acd-48d3-b4c4-4e137c8a6274)

**Comment by cryptk** [comment]
@RaHehl yes, but the issue was never about being able to set an image via the protect UI, that has obviously always worked. The question is about how to do it via this library.

**Comment by RaHehl** [comment]
@cryptk have u tried setting the lcd_message  https://github.com/uilibs/uiprotect/blob/main/src/uiprotect/data/devices.py#L946 -> https://github.com/uilibs/uiprotect/blob/main/src/uiprotect/data/devices.py#L490 with a text matching the png file name, and the set DoorbellMessageType to https://github.com/uilibs/uiprotect/blob/main/src/uiprotect/data/types.py#L412 IMAGE ?
